CombatTagPlus/src/main/java/net/minelink/ctplus/task/TagUpdateTask.java

```diff
@@ -51,6 +51,9 @@ public void run() {
             if (!plugin.getSettings().getUntagMessage().isEmpty()) {
                 player.sendMessage(plugin.getSettings().getUntagMessage());
             }
+
+            plugin.getTagManager().untag(player.getUniqueId());
```

Comment:
I find it counter intuitive to fire an untag event in a getTag method. tag and untag are the state change verbs of TagManager, untag mirroring the event firing pattern already used in tag. Also, you don't really want your getter's to have a bunch of side effects.